helmfile/charts/prometheus-alerts/files/disk-perf.yaml

```diff
@@ -0,0 +1,39 @@
+groups:
+  - name: disk-performance
+    rules:
+      - alert: DiskWaitTimeHigh
+        annotations:
+          description: Disk {{ $labels.device }} Wait Time on {{ $labels.instance }} is {{ $value }}, check the workload or adjust this alert
+          summary: Disk {{ $labels.device }} Wait Time is high on {{ $labels.instance }}
+          #documentation: https://example.org/something-wrong
+          #message: ""
+          severity_level: warning
+          storage_type: local
+        expr: |
+          (
+          rate(node_disk_read_time_seconds_total{job="node-exporter"}[1m])
```

Comment:
Is there also an equivalent metric for write?

Reply:
Yes, added https://github.com/elastisys/compliantkubernetes-apps/pull/1799/commits/a66b05d92dc8c0b44718c6739a406febd32c332a